tests/test_status.py

```diff
@@ -192,27 +297,4 @@ def test_intermediates(self):
                         ),
                     ],
                 ],
-            )
-
-    def test_parent_derive_after_intermediate(self):
-        self.status2.intermediate('interm1')
-        with StatusReader(self.status1.filepath) as status:
-            status3 = self.status1.derive()
-            status3.write('write3')
-            time.sleep(0.1)
-            self.assertEqual(status, ['write1', ['write2'], ['write3']])
-
-    def test_derive_after_intermediate(self):
-        self.status2.intermediate('interm1')
-        with StatusReader(self.status1.filepath) as status:
-            status3 = self.status2.derive()
-            status3.write('write3')
-            time.sleep(0.1)
-            self.assertEqual(status, ['write1', ['write2', ['write3']]])
-
-    def test_parent_write_after_intermediate(self):
-        self.status2.intermediate('interm1')
-        with StatusReader(self.status1.filepath) as status:
-            self.status1.write('write3')
-            time.sleep(0.1)
-            self.assertEqual(status, ['write1', ['write2'], 'write3'])
```

Comment:
Removed for now, as it seems infeasible without introducing extra complexities, but would be nice to have if a nice solution comes up.